website/pages/docs/enterprise/lease-count-quotas/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+page_title: Vault Enterprise Lease Count Quotas
+sidebar_title: Lease Count Quotas
+description: |-
+  Vault Enterprise features a mechanism to create lease count quotas.
+---
+
+# Lease Count Quotas
+
+Vault features an extension to resource quotas that allows operators to enforce
+limits on how many leases are created. For a given lease count quota, if the
+number of leases in the cluster hits the configured limit, `max_leases`, additional
+lease creations will be forbidden for all clients until a lease has been revoked
+or has expired.
```

Comment:
Should we show an example of the error?  I think that's something not shown elsewhere in the documentation.

Reply:
Sure thing 👍

Reply:
@mgritter do you think a JSON snippet of the exact response is helpful? Do we do this in other concepts/enterprise docs?

Reply:
I don't know, but I would like a search for the error text to end up at some appropriate document.

Reply:
FYI:  I added a testing section in the learn tutorial to show what happens when the quota has been reached. ([PR](https://github.com/hashicorp/learn/pull/1780))